Messages/index.js

```diff
@@ -114,6 +114,24 @@ const buildSimpleHandlers = () => {
           await message.react('🤬')
         ])
       }
+    ],
+    [
+      (text) => text === '/tusbomb',
+      async (message) => {
+        const tus = [
```

Comment:
change to read 5 random messages from #Tus Moments Museum

Reply:
sooooon :tm: